crates/cargo-rm/src/bin/cargo/commands/rm.rs

```diff
@@ -69,9 +49,92 @@ impl RmArgs {
     }
 }
 
-fn exec(args: &RmArgs) -> CargoResult<()> {
-    let manifest_path = if let Some(ref pkgid) = args.pkgid {
-        let pkg = manifest_from_pkgid(args.manifest_path.as_deref(), pkgid)?;
+pub fn cli() -> clap::Command<'static> {
+    clap::Command::new("rm")
+        .setting(clap::AppSettings::DeriveDisplayOrder)
+        .about("Remove a dependency from a Cargo.toml manifest file")
+        .args([
+            clap::Arg::new("dependencies")
+                .action(clap::ArgAction::Append)
+                .required(true)
+                .multiple_values(true)
+                .takes_value(true)
+                .value_name("DEP_ID")
+                .help("Dependencies to be removed"),
+            clap::Arg::new("manifest_path")
+                .long("manifest-path")
+                .takes_value(true)
+                .value_name("PATH")
+                .help("Path to the manifest to remove a dependency from"),
+            clap::Arg::new("pkg_id")
+                .short('p')
+                .long("package")
+                .help("Package to remove from")
+                .takes_value(true)
+                .value_name("PKGID"),
+            clap::Arg::new("dry_run")
+                .long("dry-run")
+                .help("Don't actually write the manifest"),
+            clap::Arg::new("quiet")
+                .short('q')
+                .long("quiet")
+                .help("Do not print any output in case of success"),
+        ])
+        .next_help_heading("SECTION")
+        .args([
+            clap::Arg::new("dev")
+                .short('D')
+                .long("dev")
+                .help("Remove as development dependency")
+                .conflicts_with("build")
+                .group("section"),
+            clap::Arg::new("build")
+                .short('B')
+                .long("build")
+                .help("Remove as build dependency")
+                .conflicts_with("dev")
+                .group("section"),
+            clap::Arg::new("target")
+                .long("target")
+                .help("Remove as dependency from the given target platform")
+                .takes_value(true)
+                .value_name("TARGET")
+                .value_parser(clap::builder::NonEmptyStringValueParser::new()),
+        ])
+}
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+enum UnstableOptions {}
+
+pub fn exec(_config: &mut Config, args: &ArgMatches) -> CliResult {
+    let crates = args.get_many("dependencies").unwrap().collect();
+    let dev = args.contains_id("dev");
+    let build = args.contains_id("build");
+    let target = args.get_one("target").cloned();
+    let manifest_path = args.get_one("manifest_path").cloned();
+    let pkg_id = args.get_one("pkg_id");
+    let dry_run = args.contains_id("dry_run");
+    let quiet = args.contains_id("quiet");
```

Comment:
Please either use `is_present` or update the flags to `ArgAction::SetTrue`.  Part of the intention of renaming `is_present` to `contains_id` is so that people can easily identify presence-check calls to update when porting from the old behavior to the new behavior